Vagrantfile

```diff
@@ -1,12 +1,20 @@
 # -*- mode: ruby -*-
 # vi: set ft=ruby :
+
+$script = <<SCRIPT
+COURSE=#{ENV['course']}
+sudo chown -R www-data /tmp/edx/$COURSE/
+cd /edx/app/edxapp/edx-platform
+sudo -u www-data /edx/bin/python.edxapp ./manage.py cms --settings=aws import /edx/var/edxapp/data /tmp/edx/$COURSE
```

Comment:
I think we want to use the import command from lms. It appears that the cms import command is for the contentstore app, whereas the lms import is for courseware...

Reply:
When I tried it using lms, I couldn't get the course to show up in lms or studio even after pre-creating the course in studio. However, by running the above which I found [here](https://openedx.atlassian.net/wiki/spaces/OpenOPS/pages/60227913/Managing+OpenEdX+Tips+and+Tricks), It seems to work as expected (course shows up in both lms and cms) without having to pre-create it.